src/global/Colors.js

```diff
@@ -0,0 +1,64 @@
+/*!
+ * Theme colours
+ */
+module.exports = {
+  Red: '#FF5252',
+  Pink: '#FF4081',
```

Comment:
Are the color's from our styleguide? I don't remember that we used "Pink".

Reply:
I just used a common colors file. I will remove which is not required.